File: Lista 1 sala/C/atividade06.c
// 


#include <stdio.h>

int main() {
    float valor,tempo,taxa,prestacao;

    printf("Quanto voce deve?:");
    scanf("%f", &valor);

    printf("Qual e o tempo em dias que voce atrasou:");
    scanf("%f", &tempo);

    printf("Qual o valor da taxa:");
    scanf("%f", &taxa);


    prestacao = valor + (valor*(taxa/100))*tempo;

    printf("Voce tem que pagar %.2f\n", prestacao);
    return 0;


}

File: Lista 1 sala/Python/atividade06.py
# 6)- Calcular o valor de uma prestação em atraso, usando a fórmula: PRESTAÇÃO
# <- VALOR + (VALOR * (TAXA / 100) * TEMPO).


valor=float(input("Quanto voce deve?:"))
tempo=float(input("Qual é o tempo em dias que voce atrasou:"))
taxa=float(input("Qual o valor da taxa:"))

prestação = valor + (valor*(taxa/100))*tempo

print("Voce tem que pagar", prestação)